operations/agent-flow-mixin/grizzly.jsonnet

```diff
@@ -34,4 +34,18 @@ local mixin = import './mixin.libsonnet';
     }
     for file in std.objectFields(mixin.grafanaDashboards)
   },
+
+  prometheus_rules: {
+    [file]: {
+      apiVersion: 'grizzly.grafana.com/v1alpha1',
+      kind: 'PrometheusRuleGroup',
+      metadata: {
+        folder: $.folder.metadata.name,
+        name: std.split(file, '.')[0],
+        namespace: std.split(file, '.')[0],
+      },
+      spec: mixin.prometheusAlerts[file],
+    }
+    for file in std.objectFields(mixin.prometheusAlerts)
+  },
```

Comment:
This gets rendered as following, which I'm not sure is correct yet.
```
$ jsonnet mixin.libsonnet 
...
...
   },
   "prometheusAlerts": {
      "clustering.json": {
         "rules": [
            {
               "alert": "ClusterNotConverged",
               "annotations": {
                  "message": "Cluster is not converging"
               },
               "expr": "stddev(cluster_node_peers) != 0",
               "for": "2m"
            },
            {
               "alert": "ClusterSplitBrain",
               "annotations": {
                  "message": "Cluster nodes have entered a split brain state"
               },
               "expr": "(sum without (state) (cluster_node_peers)) != count(cluster_node_info)",
               "for": "2m"
            }
         ]
      }
   }
}
```

Reply:
Have you checked to see what it renders as in the [mimir-mixin](https://github.com/grafana/mimir/tree/main/operations/mimir-mixin)?

Reply:
FWIW, it might be correct from the mixin's perspective, but then might need to be transformed to be compatible with Grizzly, like with what I have to do for the dashboards: 

https://github.com/grafana/agent/blob/fff8fb246bc87caf80d59982d5e4b5273cc63a16/operations/agent-flow-mixin/grizzly.jsonnet#L25-L35

Reply:
When calling `$ jsonnet grizzly.jsonnet`, the output contains the grizzly-specific wrapper, will check against Mimir's mixin output.